client/src/pages/Upgrade/Upgrade.tsx

```diff
@@ -35,7 +35,7 @@ const Upgrade = (): JSX.Element => {
             headerSubtitle="$5/month"
             disableButton={false}
             buttonText="Upgrade"
-            redirectUrl="/checkout"
+            priceId="price_1JDNXxKWQLQuuRfnORSql3u5"
```

Comment:
Not a great fan of hardcoding ids in client side, If the PRs have a scope of using an API for priceId, you can use that

Reply:
Updated! I just created a .ts file with list of subscriptions/priceIds so we can re-use it